example/src/components/RoomParticipant.tsx

```diff
@@ -171,6 +190,13 @@ const styles = StyleSheet.create({
     aspectRatio: 1,
     alignSelf: 'center',
   },
+  videoTrackScreencast: {
+    flex: 1,
+    aspectRatio: 1,
```

Comment:
The focused prop will have to stay either way since it is used to determine not only the style but also pin button behaviour and moving it to the parent would make it less readable in my opinion. Also managing aspect ratio in the parent isn't as simple as it may seem, since I want to keep the displayName / muted / pin icons in fixed positions whatever the aspect ratio is, and managing thet from the parent would affect that. That's also why I don't like the idea of room participant taking the entire space as it would depend on the parent aspect ratio.